### ISSUE TITLE
apps.json is now technologies.json upstream

### ISSUE THREAD

**Issue by pshevtsov** [rverton/webanalyze#36]
```
$ webanalyze -update
2020/09/04 15:24:50 app definition file updated from  https://raw.githubusercontent.com/AliasIO/Wappalyzer/master/src/apps.json

$ cat apps.json
404: Not Found
```

**Comment by thelikes** [comment]
Why did we close this? Seems like it should indeed be technologies.json, no?

**Comment by rverton** [comment]
I fixed this in 3ef937d by renaming it.

**Comment by thelikes** [comment]
Gotcha. Struggling with a `go get -u` , but cloning the repo and manually building, seems all is well. Thank you, appreciate your work and this tool!
